src/libPMacc/include/mpi/MPIReduce.hpp

```diff
@@ -52,6 +52,7 @@ struct MPIReduce
     {
         if (isMPICommInitialized)
         {
+            /* exception inside the destructor are forbidden */
```

Comment:
I added this commend because it was not clear why `NOEXEPT` was used and not the old version. We should leave this comment in for now that it is easier for me to review it later if it goes back to the mainline of PIConGPU